src/lib/containers/dataaccess/AccessRequirementTable.tsx

```diff
@@ -0,0 +1,174 @@
+import moment from 'moment'
+import React, { useMemo, useState } from 'react'
+import { Button, Table } from 'react-bootstrap'
+import SortIcon from '../../assets/icons/Sort'
+import { formatDate } from '../../utils/functions/DateFormatter'
+import { PRODUCTION_ENDPOINT_CONFIG } from '../../utils/functions/getEndpoint'
+import { useSearchAccessRequirementsInfinite } from '../../utils/hooks/SynapseAPI/dataaccess/useGetAccessRequirement'
+import { ACT_TEAM_ID } from '../../utils/SynapseConstants'
+import { ACCESS_TYPE } from '../../utils/synapseTypes'
+import {
+  AccessRequirementSearchRequest,
+  AccessRequirementSearchSort,
+} from '../../utils/synapseTypes/AccessRequirement/AccessRequirementSearch'
+import Typography from '../../utils/typography/Typography'
+import { EntityLink } from '../EntityLink'
+import IconSvg from '../IconSvg'
+import { SynapseSpinner } from '../LoadingScreen'
+import UserOrTeamBadge from '../UserOrTeamBadge'
+
+export type AccessRequirementTableProps = {
+  nameContains?: string
+  relatedProjectId?: string
+  reviewerId?: string
+  accessType?: ACCESS_TYPE
+  onCreateNewAccessRequirementClicked?: () => void
+}
+
+export default function AccessRequirementTable(
+  props: AccessRequirementTableProps,
+) {
+  const {
+    nameContains,
+    relatedProjectId,
+    reviewerId,
+    accessType,
+    onCreateNewAccessRequirementClicked,
+  } = props
+
+  const [sort, setSort] = useState<AccessRequirementSearchSort>({
+    field: 'CREATED_ON',
+    direction: 'DESC',
+  })
+
+  const searchRequest: Omit<AccessRequirementSearchRequest, 'nextPageToken'> =
+    useMemo(
+      () => ({
+        nameContains,
+        relatedProjectId,
+        reviewerId,
+        accessType,
+        sort: [sort],
+      }),
+      [nameContains, relatedProjectId, reviewerId, accessType, sort],
+    )
+
+  const { data, hasNextPage, fetchNextPage, isFetching } =
+    useSearchAccessRequirementsInfinite(searchRequest)
+
+  const accessRequirements = data?.pages.flatMap(page => page.results) ?? []
+  const onSort = (field: AccessRequirementSearchSort['field']) => {
+    if (sort.field === field) {
+      setSort({ field, direction: sort.direction === 'DESC' ? 'ASC' : 'DESC' })
+    } else {
+      setSort({ field, direction: 'DESC' })
+    }
+  }
+
+  return (
+    <div className="bootstrap-4-backport">
+      <div className="SRC-split">
+        <Typography variant="headline3" style={{ marginBottom: 0 }}>
+          Access Requirements
```

Comment:
Should this have the number of ARs on table?

Reply:
This was missed earlier so we don't have a backend service to determine this number. I brought it up with Adam, we'll backfill this feature if we determine it's necessary.